scripts/build-sentry-native.ps1

```diff
@@ -0,0 +1,76 @@
+param([switch] $Clean)
+Set-StrictMode -Version Latest
+$ErrorActionPreference = "Stop"
+
+Push-Location $PSScriptRoot/..
+try
+{
+    $submodule = 'modules/sentry-native'
+    $outDir = 'src/Sentry/Platforms/Native/runtimes'
+    $buildDir = "$submodule/build"
+    $actualBuildDir = $buildDir
+
+    $additionalArgs = @()
+    $libPrefix = 'lib'
+    $libExtension = '.a'
+    if ($IsMacOS)
+    {
+        $outDir += '/osx/native'
+        $additionalArgs += @('-D', 'CMAKE_OSX_ARCHITECTURES=arm64;x86_64')
+    }
+    elseif ($IsWindows)
+    {
+        $outDir += '/win-x64/native'
+        $additionalArgs += @('-C', 'src/Sentry/Platforms/Native/windows-config.cmake')
+        $actualBuildDir = "$buildDir/RelWithDebInfo"
+        $libPrefix = ''
+        $libExtension = '.lib'
+    }
+    elseif ($IsLinux)
+    {
+        $outDir += '/linux-x64/native'
+    }
+    else
+    {
+        throw "Unsupported platform"
+    }
+
+    git submodule update --init --recursive $submodule
+
+    if ($Clean)
+    {
+        rm -rf $buildDir
+    }
+
+    cmake `
+        -S $submodule `
+        -B $buildDir `
+        -D CMAKE_BUILD_TYPE=RelWithDebInfo `
```

Comment:
Debug info will be bundled in case of mac/linux? And on Windows we'll get dlls?

Should we upload this through Symbol Collector and avoid bundling that in the NuGet Package? It's already extremely large with all the Sentry CLI per platform

Reply:
We're compiling static libraries, therefore the debug info will only be created once the final app is compiled (with addresses relative to the final executable). 

The best we can do to reduce package size is stop packaging CLI and let it be downloaded later during user project's build, as needed. 

BTW, I've also reduced sentry CLi binary size recently (should come out with the latest release: https://github.com/getsentry/sentry-cli/pull/1794)

Reply:
it's all good, sorry I forgot about it being a static lib